DragaliaAPI.Shared/MasterAsset/MasterAsset.cs

```diff
@@ -29,4 +29,14 @@ public static class MasterAsset
     private static readonly Lazy<MasterAssetData<WeaponBodies, WeaponBody>> _WeaponBody =
         new(() => new("WeaponBody.json", x => x.Id));
     public static MasterAssetData<WeaponBodies, WeaponBody> WeaponBody => _WeaponBody.Value;
+    private static readonly Lazy<MasterAssetData<int, StoryData>> _DragonStories =
```

Comment:
i think i said to use ints as the key but I didn't realize they were split -- is it still not possible to use the enum because of non-playable units?